platform/ripple/api.go

```diff
@@ -82,18 +79,24 @@ func NormalizeTx(srcTx *Tx) (blockatlas.Tx, bool) {
 		return blockatlas.Tx{}, false
 	}
 
-	if srcTx.Payment.TransactionType != "Payment" {
+	if srcTx.Payment.TransactionType != transactionPayment {
 		return blockatlas.Tx{}, false
 	}
 
+	status := blockatlas.StatusCompleted
+	if srcTx.Meta.TransactionResult != tesSUCCESS {
```

Comment:
what other possible cases there is? to avoid defaulting to failed

Reply:
all other codes are errors, only one can be a success is the ter Codes (Retry), but another transaction going to be applied in another block

https://xrpl.org/transaction-results.html